test.py

```diff
@@ -697,6 +697,67 @@ def test_render_reference(self):
         self.assertEqual(len(lines), 2)
         self.assertIn(url, lines[1].text)
 
+class BaseV3WriterTest(unittest.TestCase):
+    '''BaseV3Writer tests'''
+
+    def setUp(self):
+        xml2rfc.log.quiet = True
+        path = 'tests/input/elements.xml'
+        self.parser = xml2rfc.XmlRfcParser(path,
+                                           quiet=True,
+                                           options=default_options,
+                                           **options_for_xmlrfcparser)
+        self.xmlrfc = self.parser.parse()
+        self.writer = BaseV3Writer(self.xmlrfc, quiet=True)
+
+    def test_validate_draft_name(self):
+        # Draft with only docName
+        self.assertTrue(self.writer.validate_draft_name())
+
+        # RFC
+        rfc = lxml.etree.fromstring('''
+<rfc
+    number="9280"
+    docName = "draft-ietf-foobar-23"
+    ipr="trust200902"
+    submissionType="editorial"
+    category="info">
+    <front>
+        <seriesInfo name="Internet-Draft" value="draft-ietf-foobar-23" />
```

Comment:
RFCs don't have this - they have an rfc series info entry instead

Reply:
Updated.